src/reactviews/pages/QueryResult/table/plugins/headerFilter.plugin.ts

```diff
@@ -60,10 +61,16 @@ export class HeaderFilter<T extends Slick.SlickData> {
 
     private _eventManager = new EventManager();
     private queryResultState: QueryResultState;
+    private gridId: string;
 
-    constructor(theme: ColorThemeKind, queryResultState: QueryResultState) {
+    constructor(
+        theme: ColorThemeKind,
+        queryResultState: QueryResultState,
+        gridId: string,
```

Comment:
you can also do `constructor(private gridId: string)` as shorthand for all these constructor changes